common/src/Microsoft.Azure.IIoT.Hub.Module.Client/src/Default/IoTSdkFactory.cs

```diff
@@ -720,10 +302,11 @@ internal sealed class IoTSdkLogger : EventSourceSerilogSink {
             public const string EventSource = "Microsoft-Azure-Devices-Device-Client";
         }
 
-        private readonly TimeSpan _timeout;
         private readonly TransportOption _transport;
-        private readonly IotHubConnectionStringBuilder _cs;
+        private readonly IotHubConnectionStringBuilder _deviceClientCs;
```

Comment:
is it realistic that we have configured both _deviceClientCs and _mqttClientCs at the same time? if so, who wins?

Reply:
If both are configured constructor throws exception.